packages/ui-components/src/icons.tsx

```diff
@@ -42,11 +42,11 @@ export const importIcon = new LabIcon({
   svgstr: importSvg
 });
 export const airflowIcon = new LabIcon({
-  name: 'elyra:airflow',
+  name: 'elyra:APACHE_AIRFLOW',
```

Comment:
The displayable values should come from the server.  If this were a straight value, it would be fine to change, but, unfortunately, I believe I remember code that uses `elyra:` as a _prefix_ and, in those cases, I think it would be better wait for the proper means of dealing with icons.